### PR TITLE
guide/KV-stores-namespaces

### LINKED ISSUE #15: Research basic API usage with CloudFlare KV Stores
Research basic API usage with CloudFlare KV Stores and create a basic usage guide.

https://developers.cloudflare.com/workers/learning/how-kv-works/

KV Stores
- Easiest way to work with via api. We'll want to read/write directly to KV stores from Svelte API routes and CLI actions

### PR DESCRIPTION
**Pull request by kimberlym4488**
- [ ] Markdown detailing CF worker publish creating KV namespaces, route, and binding. 
- [ ] Wrangler CLI commands and JS Crud methods for keys/values included.
- [ ] kvtest, app.d.ts, hooks file set up included in guide

Ready for review!

Resolves #15

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -16,5 +16,8 @@ module.exports = {
 		browser: true,
 		es2017: true,
 		node: true
+	},
+	globals: {
+		FILES: readonly
```
Comment (ac524) [inline .eslintrc.cjs:21]: These changes should be removed. They won't be relevant to our usage.

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,4 @@ node_modules
 .env
 .env.*
 !.env.example
+wrangler.toml
```
Comment (ac524) [inline .gitignore:10]: Let's remove this for now. I'll need this file committed for the CICD pipeline.

**Review by ac524 — CHANGES_REQUESTED** [review]
@kimberlym4488 The table of contents is great. I'm not sure how much value the images and specific info about the FILES namespace are adding in the guide, so I think we should take that out for now.

Also, I think there are a number of images you've added in the assets folder I don't see being used, so those should be cleaned up.

**Comment by kimberlym4488** [comment]
@ac524  should be all set!

**Review by ac524 — APPROVED** [review]

**Pull request merged**